apiv2/schema/schema.yaml

```diff
@@ -1545,69 +1581,6 @@ classes:
         - api_https_prefix
         range: string
         required: true
-  PerSectionParameters:
-    name: PerSectionParameters
-    annotations:
-      plural:
-        tag: plural
-        value: PerSectionParameters
-    description: Record how frames get mapped to TiltSeries
-    from_schema: cdp-dataset-config
-    mixins:
-    - IDMixin
-    attributes:
-      frame:
-        name: frame
-        from_schema: cdp-dataset-config
-        inverse: Frame.per_section_parameters
-        range: Frame
-        required: true
-      tiltseries:
-        name: tiltseries
-        from_schema: cdp-dataset-config
-        inverse: Tiltseries.per_section_parameters
-        range: Tiltseries
-        required: true
-      z_index:
-        name: z_index
-        description: z-index of the frame in the tiltseries
-        from_schema: cdp-dataset-config
-        exact_mappings:
-        - per_section_z_index
-        range: integer
-        required: true
-        minimum_value: 0
-      defocus:
-        name: defocus
-        description: defocus amount
-        from_schema: cdp-dataset-config
-        exact_mappings:
-        - per_section_defocus
-        range: float
-        minimum_value: 100
-        maximum_value: 100
-        unit:
-          symbol: µm
-          descriptive_name: microns
-      astigmatism:
-        name: astigmatism
-        description: Astigmatism amount for this frame
-        from_schema: cdp-dataset-config
-        exact_mappings:
-        - per_section_astigmatism
-        range: float
-      astigmatic_angle:
-        name: astigmatic_angle
-        description: Angle of ast
-        from_schema: cdp-dataset-config
-        exact_mappings:
-        - per_section_astigmatic_angle
-        range: float
-        minimum_value: -180
-        maximum_value: 180
-        unit:
-          symbol: °
-          descriptive_name: degrees
   PerSectionAlignmentParameters:
     name: PerSectionAlignmentParameters
```

Comment:
- The `in_plane_rotation` field will be a collection of 4 float values. Should we make it a vector or a string? 
- The `beam_tilt` field is not going to be supported for now. 
- We should add support for the `is_canonical` boolean field and the float `volume_x_rotation`.

Reply:
Supporting vectors and other *strongly typed* json fields is pretty doable in the API - we already support 2d arays of floats, so supporting 1d arrays is easy to handle. I haven't tried to add complex types to the client before though, so that could get interesting. Do you think `in_plane_rotation` would be best represented as a list of floats?

Reply:
@uermel Could we have your thoughts on what is the best way to represent `in_plane_rotation` for the end user?

Reply:
Decision: This will be a 2d rotation matrix, basically `list[list[float]]`